packages/core/src/services/warehouses/WarehousesOrdersService.ts

```diff
@@ -270,6 +270,9 @@ export class WarehousesOrdersService
 			orderNumber: await this.getNextOrderNumber(warehouseId),
 			orderType,
 			...(options.autoConfirm ? { isConfirmed: true } : {}),
+			...(options.autoConfirm
+				? { warehouseStatus: OrderWarehouseStatus.ReadyForProcessing }
+				: { warehouseStatus: OrderWarehouseStatus.NoStatus }),
```

Comment:
After discussed this, I understand that this is not in the scope of the current task, but she decided to put it and fix a bug produced by feature for autoConfirm. So I suggest to revert this, then she to create issue with video which show the problem.

Reply:
@AlishMekliov931 @tsvetelina-e-y I don't mind if that PR will fix some other issue, we can always add video to the same PR or ticket etc. However, I just not sure about logic of that code above, that's why asked you to review. Is it OK to do it that way and it fix some issues? If yes, all fine, we can merge

Reply:
@evereq I will revert this changes with changing status of order for 1) as @AlishMekliov931 explained,  isConfirmed property is actually not related to confirmation from merchant side, but from client 2) it is not related with the issue.